pyppeteer/browser_fetcher.py

```diff
@@ -44,10 +44,10 @@ class RevisionInfo(TypedDict):
     revision: str
 
 
-class BrowserOptions(TypedDict, total=False):
-    platform: Platforms
-    path: Path
-    host: str
+# class BrowserFetcherOptions(TypedDict, total=False):
+#     platform: Platforms
+#     path: Path
+#     host: str
```

Comment:
```suggestion
```
Or is reserved for future?

Reply:
Nope, doesn't make sense to use it in the future. All places where this occurs have been expanded to regular args.